IDE/GCC-ARM/Makefile.common

```diff
@@ -80,6 +80,7 @@ SRC_C += ../../wolfcrypt/benchmark/benchmark.c
 endif
 
 # WOLFSSL TLS FILES
+SRC_C += ../../src/bio.c
```

Comment:
Technically bio.c is not needed, since it is included via ssl.c. There are some macros protections to allow this, but you may want to make sure `WOLFSSL_IGNORE_FILE_WARN` is defined in the user_settings.h.

Reply:
ok Adding to user_settings